.changeset/chilly-trainers-type.md

```diff
@@ -0,0 +1,7 @@
+---
+"@nextui-org/button": patch
+"@nextui-org/theme": patch
+---
+
+Fix #1626 The radius now works, and the border is displayed correctly in the buttonGroup.
```

Comment:
Can you describe the change more precisely, "now works" does not tell me anything?

Reply:
sorry, @jguddas  can you check this again? thank you.